dashboard/http_server_head.py

```diff
@@ -109,11 +109,13 @@ async def get_index(self, req) -> aiohttp.web.FileResponse:
                 "This error message is harmless and can be ignored. "
                 f"Error: {e}"
             )
-        return aiohttp.web.FileResponse(
+        resp = aiohttp.web.FileResponse(
             os.path.join(
                 os.path.dirname(os.path.abspath(__file__)), "client/build/index.html"
             )
         )
+        resp.headers["Cache-Control"] = "no-cache"
```

Comment:
Is there any risk of the key `"Cache-Control"` not being present?

Reply:
nope, the default behavior is determined by browser when it's not set.